### PR TITLE
chore(css): Move CSS examples - Basic shapes, filter effects, module template, paint API

### PR DESCRIPTION
**Pull request by bsmth**
### Description

As part of an initiative to reduce repo maintenance & code duplication, this PR moves some external examples into content. This PR converts the following macros to live samples:

```
{{EmbedGHLiveSample("css-examples/cssom/modify-rule.html", '100%', 120)}}
{{EmbedGHLiveSample("css-examples/houdini/css_painting_api/example-boxbg.html", '100%', 400)}}
{{EmbedGHLiveSample("css-examples/shapes/overview/circle.html", '100%', 720)}}
{{EmbedGHLiveSample("css-examples/shapes/basic-shape/ellipse.html", '100%', 800)}}
{{EmbedGHLiveSample("css-examples/shapes/basic-shape/ellipse-keywords.html", '100%', 800)}}
{{EmbedGHLiveSample("css-examples/flexbox/alignment/intro.html", '100%', 700)}}
{{EmbedGHLiveSample("css-examples/modules/filters.html", '100%', 420)}}
{{EmbedGHLiveSample("css-examples/shapes/overview/circle.html", '100%', 720)}}
{{EmbedGHLiveSample("css-examples/houdini/css_painting_api/example-boxbg.html", '100%', 400)}}
```

### Motivation

Motivation is described in the project issue.

### Related issues and pull requests

- https://github.com/mdn/mdn/issues/597

### CODE REVIEW

File: files/en-us/web/api/css_painting_api/index.md [hunk 1]
```diff
@@ -36,15 +36,17 @@ The API defines a {{domxref('worklet')}} that can be used to programmatically ge
 
 ## Examples
 
-The following example creates a list of items with a background image that rotates between three different colors and three widths. In a supporting browser you will see something like the image below.
+The following example creates a list of items with a background image that rotates between three different colors and three widths.
+In [a supporting browser](#browser_compatibility) you will see something like the image below.
 
 ![The width and color of the background image changes based on the custom properties](guide/boxbg.png)
```
Comment (estelle) [inline files/en-us/web/api/css_painting_api/index.md:42]: This image is not rendering

File: files/en-us/learn/css/css_layout/responsive_design/index.md [hunk 2]
```diff
@@ -135,8 +135,71 @@ In the example below the flex items will each take an equal amount of space in t
 }
 ```
 
-> [!NOTE]
-> As an example, we have built a simple responsive layout above using flexbox. We use a breakpoint to switch to multiple columns when the screen grows, and limit the size of the main content with {{cssxref('max-width')}}: [example](https://mdn.github.io/css-examples/learn/rwd/flex-based-rwd.html), [source code](https://github.com/mdn/css-examples/blob/main/learn/rwd/flex-based-rwd.html).
+Here's how we could use flexbox with a media query for responsive design.
+You can click "Play" to view and edit the source code in the MDN Playground:
```
Comment (estelle) [inline files/en-us/learn/css/css_layout/responsive_design/index.md:139]:
```suggestion
```

File: files/en-us/learn/css/css_layout/responsive_design/index.md [hunk 3]
```diff
@@ -135,8 +135,71 @@ In the example below the flex items will each take an equal amount of space in t
...
+    margin-right: 5%;
+  }
+
+  .col2 {
+    flex: 2;
+  }
+}
+```
+
+{{EmbedLiveSample("flex-based-rwd", "", "550px")}}
```
Comment (estelle) [inline files/en-us/learn/css/css_layout/responsive_design/index.md:202]:
```suggestion
{{EmbedLiveSample("flex-based-rwd", "", "550px")}}

Resize your screen. The layout will change when the size of the above example crosses the 600px width threshold.
```
Or something like that. Only recommend Play when needed, not for each example. This goes for all the occurrences of 'You can click "Play" to view and edit the source code in the MDN Playground:"'

File: files/en-us/learn/css/css_layout/responsive_design/index.md [hunk 4]
```diff
@@ -149,8 +212,64 @@ In CSS grid layout the `fr` unit allows the distribution of available space acro
 }
 ```
 
-> [!NOTE]
-> The grid layout version is even simpler as we can define the columns on the .wrapper: [example](https://mdn.github.io/css-examples/learn/rwd/grid-based-rwd.html), [source code](https://github.com/mdn/css-examples/blob/main/learn/rwd/grid-based-rwd.html).
+Here's how we could use grid layout with a media query for responsive design.
+You can click "Play" to view and edit the source code in the MDN Playground:
```
Comment (estelle) [inline files/en-us/learn/css/css_layout/responsive_design/index.md:216]:
```suggestion
```

File: files/en-us/mdn/writing_guidelines/page_structures/page_types/css_module_landing_page_template/index.md [hunk 5]
```diff
@@ -62,10 +62,13 @@ This should ideally be one or two short sentences.
 
 ## NameOfTheModule in action
 
-In this section, include an interactive example of the module that helps to demonstrate the usefulness or the power of various properties provided by this module. The purpose of this section is to demonstrate a few use cases and to create interest and curiosity in the mind of the readers learning about this module.
+In this section, include an example that helps to demonstrate the usefulness or the power of various properties provided by this module.
```
Comment (estelle) [inline files/en-us/mdn/writing_guidelines/page_structures/page_types/css_module_landing_page_template/index.md:65]:
```suggestion
In this section, include an example that helps demonstrate the usefulness or the power of various properties provided by this module.
```

File: files/en-us/web/api/css_object_model/using_dynamic_styling_information/index.md [hunk 6]
```diff
@@ -16,32 +16,27 @@ To get to the `style` objects from the `document`, you can use the {{domxref("Do
 
 ## Modify a stylesheet rule with CSSOM
 
-In this example the background of the page is set to red using CSS. The JavaScript then accesses the property using the CSSOM and changes the background to blue.
+In this example the background of the page is set to red using CSS. The JavaScript then accesses the property using the CSSOM and changes the background to `cornflowerblue`.
```
Comment (estelle) [inline files/en-us/web/api/css_object_model/using_dynamic_styling_information/index.md:19]:
```suggestion
In this example the background of the page is set to `red` using CSS. The JavaScript then accesses the property using the CSSOM and changes the background to `cornflowerblue`.
```

File: files/en-us/web/api/css_painting_api/index.md [hunk 7]
```diff
@@ -53,22 +55,21 @@ registerPaint(
...
       return { alpha: true };
     }
-
     /*
-     use this function to retrieve any custom properties (or regular properties, such as 'height')
-     defined for the element, return them in the specified array
-  */
+      Retrieve any custom properties (or regular properties,
+      such as 'height') defined for the element, and return
+      them in the specified array.
```
Comment (estelle) [inline files/en-us/web/api/css_painting_api/index.md:61]: I realize this is not part of this PR, but should this read:

```suggestion
      them as an array.
```

File: files/en-us/web/css/basic-shape/ellipse/index.md [hunk 8]
```diff
@@ -42,15 +42,76 @@ An ellipse is essentially a squashed circle and so `ellipse()` acts in a very si
 
 ### Basic ellipse() example
 
-This example shows an ellipse with an x radius of 40%, a y radius of 50% and the position being left. This means that the center of the ellipse is on the left edge of the box giving us a half ellipse shape to wrap our text around. You can change these values to see how the ellipse changes.
+This example shows an ellipse with an x radius of 40%, a y radius of 50% and the position being left. This means that the center of the ellipse is on the left edge of the box giving us a half ellipse shape to wrap our text around.
```
Comment (estelle) [inline files/en-us/web/css/basic-shape/ellipse/index.md:45]: also, not part of this PR, but should we write:

```suggestion
This example shows an ellipse that is floated left that has a horizontal radius of 40%, a vertical radius of 50%, and a left position. This means that the center of the ellipse is on the left edge of the box giving us a half ellipse shape to wrap our text around.
```

File: files/en-us/web/css/css_filter_effects/index.md [hunk 9]
```diff
@@ -13,11 +13,237 @@ The properties in the **CSS filter effects** module let you define a way of proc
 
 ### Filter effects in action
 
-Play with the various sliders to apply filter effects to the image below:
+Play with the various sliders to apply filter effects to the image below.
+You can click "Play" to view and edit the code in the MDN Playground:
```
Comment (estelle) [inline files/en-us/web/css/css_filter_effects/index.md:17]:
```suggestion
```
definitely do not send people to Play for this example. They can play with all the code using the ranges provided.

**Review by estelle — COMMENTED** [review]
Until Play supports saving work, for learn pages with examples that build on previous examples, opt to encourage creating and saving files rather than play

File: files/en-us/learn/css/css_layout/responsive_design/index.md (hunk 3 above)
Comment (bsmth) [reply files/en-us/learn/css/css_layout/responsive_design/index.md:202]: Okay, no objections to removing. I'm usually adding "click Play" if there's hidden code blocks that would be useful for the reader to see expanded, or if editing the source is recommended in prose.

File: files/en-us/learn/css/css_layout/media_queries/index.md [hunk 10]
```diff
@@ -303,54 +479,13 @@ article {
...
 
-[Open step 1](https://mdn.github.io/css-examples/learn/media-queries/step1.html) in the browser, or [view the source](https://github.com/mdn/css-examples/blob/main/learn/media-queries/step1.html).
-
-**If you want to follow on and implement this example as we go, make a local copy of [step1.html](https://github.com/mdn/css-examples/blob/main/learn/media-queries/step1.html) on your computer.**
+{{EmbedLiveSample("walkthrough", "", "600px")}}
 
 From this point, start to drag the Responsive Design Mode view wider until you can see that the line lengths are becoming quite long, and we have space for the navigation to display in a horizontal line. This is where we'll add our first media query. We'll use ems, as this will mean that if the user has increased their text size, the breakpoint will happen at a similar line-length but wider viewport, than someone with a smaller text size.
 
-**Add the below code into the bottom of your step1.html CSS.**
+Add the following CSS to your styles:
```
[the remark this answers is not in the document]
Comment (bsmth) [reply files/en-us/learn/css/css_layout/media_queries/index.md:488]: Yeah this is much better

File: files/en-us/learn/css/css_layout/media_queries/index.md [hunk 11]
```diff
@@ -78,34 +78,58 @@ The following media query will only set the body to 12pt if the page is printed.
...
 
-[Open this example](https://mdn.github.io/css-examples/learn/media-queries/max-width.html) in the browser, or [view the source](https://github.com/mdn/css-examples/blob/main/learn/media-queries/max-width.html).
+```html hidden live-sample___max-width
+<p>
+  One November night in the year 1782, so the story runs, two brothers sat over
+  their winter fire in the little French town of Annonay, watching the grey
+  smoke-wreaths from the hearth curl up the wide chimney. Their names were
+  Stephen and Joseph Montgolfier, they were papermakers by trade, and were noted
+  as possessing thoughtful minds and a deep interest in all scientific knowledge
+  and new discovery.
```
[the remark this answers is not in the document]
Comment (bsmth) [reply files/en-us/learn/css/css_layout/media_queries/index.md:128]: This is apparently an excerpt from "BALLOONS AIRSHIPS AND FLYING MACHINES"

https://www.gutenberg.org/files/54799/54799-h/54799-h.htm

Should we really alter it?

File: files/en-us/web/api/css_painting_api/index.md (hunk 1 above)
Comment (bsmth) [reply files/en-us/web/api/css_painting_api/index.md:42]: Ah, it's not available on the preview, but it's working locally:

image

And on prod: https://developer.mozilla.org/en-US/docs/Web/API/CSS_Painting_API#examples

File: files/en-us/learn/css/css_layout/media_queries/index.md (hunk 11 above)
[the remark this answers is not in the document]
Comment (estelle) [reply files/en-us/learn/css/css_layout/media_queries/index.md:128]: Too funny. It's old English. 1905. In 1905, in Europe, they did not have hundreds of millions of people, much less in 1782. We should likely come up with a different example, but don't need to do that today. Thanks for finding the reference.

**Review by estelle — APPROVED** [review]
Thanks! 🎉

**Comment by bsmth** [comment]
Thank you!
